### DISCUSSION TITLE
VSCode extension doesn't resolve non-public function access properly

### DISCUSSION THREAD

**Discussion by austinkline** [onflow/flow#1225] [Developer Experience] [answered]
If I make a function in one contract that is imported by another with `access(account)` instead of `pub`, VSCode doesn't think I have access and shows an error instead

**Comment by btspoony** [comment, marked as answer]
Related Issues:

- [x] https://github.com/onflow/vscode-cadence/issues/75
- [x] https://github.com/onflow/vscode-cadence/issues/199
